### ISSUE TITLE
Dense Matrix Inverse based on SVD

### ISSUE THREAD

**Issue by deepaky7131** [libMesh/libmesh#1786]
Implemented a new function for taking inverse of a dense matrix using svd

**Comment by roystgnr** [comment]
Good idea.

If you're working on this, could you expose access to the singular values and vectors while you're at it?  I've got a use case that could greatly benefit.

**Comment by jwpeterson** [comment]
@roystgnr Doesn't DenseMatrix::svd() already do this?
```
  void svd(DenseVector<Real> & sigma,
           DenseMatrix<Number> & U,
           DenseMatrix<Number> & VT);
```

**Comment by deepaky7131** [comment]
Yes, I just wrote a template, that multiplies V*sigmainverse*U_transpose. I
will be using the function for a kernel that i am working in Peridynamics
module.

**Comment by roystgnr** [comment]
Ha, you're right!  Thanks!
